airflow/dags/operations/kernel_changes_operations.py

```diff
@@ -354,9 +360,9 @@ def try_register_documents_renditions(
             article = article_rendition_factory(document, data)
             article.save()
         except models.Article.DoesNotExist:
-            logging.info(
-                'Could not possible save rendition for document, probably '
-                'document %s isn\'t in OPAC database.' % document
+            logging.error(
+                "Could not save renditions for document '%s' because "
+                "it does not exists in website database." % document
```

Comment:
O correto seria *it does not exist*

Reply:
@gustavofonseca por gentileza você poderia revisitar o PR?

Reply:
Sim, estou revisando =]